[31m[App] 5557   - [39m6/24/2024, 8:13:17 PM [31m  ERROR[39m  [31mDBError: select "users".* from "users" where "0" like $1 limit $2 - column "0" does not exist
    at wrapError (/Users/henry/Projects/personal/vms/node_modules/db-errors/lib/dbErrors.js:19:14)
    at handleExecuteError (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:1140:32)
    at TableDataBuilder.execute (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:449:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at GetForTable.execute (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5251:24)
    at TableDataBuilder.getForTable (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5263:24)
    at UsersService.findPaging (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5365:16)[39m
[31m[App] 14661  - [39m6/24/2024, 9:46:20 PM [31m  ERROR[39m [33m[NestApplication][39m [31mError: listen EADDRINUSE: address already in use :::3333[39m - {"stack":[null]}
[31m[App] 14661  - [39m6/24/2024, 9:46:20 PM [31m  ERROR[39m  [31munhandledRejection: listen EADDRINUSE: address already in use :::3333
Error: listen EADDRINUSE: address already in use :::3333
    at Server.setupListenHandle [as _listen2] (node:net:1898:16)
    at listenInCluster (node:net:1946:12)
    at Server.listen (node:net:2044:7)
    at ExpressAdapter.listen (/Users/henry/Projects/personal/vms/node_modules/@nestjs/platform-express/adapters/express-adapter.js:95:32)
    at /Users/henry/Projects/personal/vms/node_modules/@nestjs/core/nest-application.js:180:30
    at new Promise (<anonymous>)
    at NestApplication.listen (/Users/henry/Projects/personal/vms/node_modules/@nestjs/core/nest-application.js:170:16)
    at startServer (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:885:21)[39m - {"error":{"code":"EADDRINUSE","errno":-48,"syscall":"listen","address":"::","port":3333},"stack":"Error: listen EADDRINUSE: address already in use :::3333\n    at Server.setupListenHandle [as _listen2] (node:net:1898:16)\n    at listenInCluster (node:net:1946:12)\n    at Server.listen (node:net:2044:7)\n    at ExpressAdapter.listen (/Users/henry/Projects/personal/vms/node_modules/@nestjs/platform-express/adapters/express-adapter.js:95:32)\n    at /Users/henry/Projects/personal/vms/node_modules/@nestjs/core/nest-application.js:180:30\n    at new Promise (<anonymous>)\n    at NestApplication.listen (/Users/henry/Projects/personal/vms/node_modules/@nestjs/core/nest-application.js:170:16)\n    at startServer (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:885:21)","rejection":true,"date":"Mon Jun 24 2024 21:46:20 GMT+0700 (Indochina Time)","process":{"pid":14661,"uid":501,"gid":20,"cwd":"/Users/henry/Projects/personal/vms","execPath":"/Users/henry/.nvm/versions/node/v20.13.1/bin/node","version":"v20.13.1","argv":["/Users/henry/.nvm/versions/node/v20.13.1/bin/node","/Users/henry/Projects/personal/vms/node_modules/@nx/js/src/executors/node/node-with-require-overrides"],"memoryUsage":{"rss":131629056,"heapTotal":71008256,"heapUsed":42241512,"external":4365011,"arrayBuffers":141457}},"os":{"loadavg":[2.32080078125,3.01708984375,3.1904296875],"uptime":39672},"trace":[{"column":16,"file":"node:net","function":"Server.setupListenHandle [as _listen2]","line":1898,"method":"setupListenHandle [as _listen2]","native":false},{"column":12,"file":"node:net","function":"listenInCluster","line":1946,"method":null,"native":false},{"column":7,"file":"node:net","function":"Server.listen","line":2044,"method":"listen","native":false},{"column":32,"file":"/Users/henry/Projects/personal/vms/node_modules/@nestjs/platform-express/adapters/express-adapter.js","function":"ExpressAdapter.listen","line":95,"method":"listen","native":false},{"column":30,"file":"/Users/henry/Projects/personal/vms/node_modules/@nestjs/core/nest-application.js","function":null,"line":180,"method":null,"native":false},{"column":null,"file":null,"function":"new Promise","line":null,"method":null,"native":false},{"column":16,"file":"/Users/henry/Projects/personal/vms/node_modules/@nestjs/core/nest-application.js","function":"NestApplication.listen","line":170,"method":"listen","native":false},{"column":21,"file":"/Users/henry/Projects/personal/vms/dist/apps/api/main.js","function":"startServer","line":885,"method":null,"native":false}]}
[31m[App] 14661  - [39m6/24/2024, 9:46:20 PM [31m  ERROR[39m  [31munhandledRejection: listen EADDRINUSE: address already in use :::3333
Error: listen EADDRINUSE: address already in use :::3333
    at Server.setupListenHandle [as _listen2] (node:net:1898:16)
    at listenInCluster (node:net:1946:12)
    at Server.listen (node:net:2044:7)
    at ExpressAdapter.listen (/Users/henry/Projects/personal/vms/node_modules/@nestjs/platform-express/adapters/express-adapter.js:95:32)
    at /Users/henry/Projects/personal/vms/node_modules/@nestjs/core/nest-application.js:180:30
    at new Promise (<anonymous>)
    at NestApplication.listen (/Users/henry/Projects/personal/vms/node_modules/@nestjs/core/nest-application.js:170:16)
    at startServer (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:885:21)[39m - {"error":{"code":"EADDRINUSE","errno":-48,"syscall":"listen","address":"::","port":3333},"stack":"Error: listen EADDRINUSE: address already in use :::3333\n    at Server.setupListenHandle [as _listen2] (node:net:1898:16)\n    at listenInCluster (node:net:1946:12)\n    at Server.listen (node:net:2044:7)\n    at ExpressAdapter.listen (/Users/henry/Projects/personal/vms/node_modules/@nestjs/platform-express/adapters/express-adapter.js:95:32)\n    at /Users/henry/Projects/personal/vms/node_modules/@nestjs/core/nest-application.js:180:30\n    at new Promise (<anonymous>)\n    at NestApplication.listen (/Users/henry/Projects/personal/vms/node_modules/@nestjs/core/nest-application.js:170:16)\n    at startServer (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:885:21)","rejection":true,"date":"Mon Jun 24 2024 21:46:20 GMT+0700 (Indochina Time)","process":{"pid":14661,"uid":501,"gid":20,"cwd":"/Users/henry/Projects/personal/vms","execPath":"/Users/henry/.nvm/versions/node/v20.13.1/bin/node","version":"v20.13.1","argv":["/Users/henry/.nvm/versions/node/v20.13.1/bin/node","/Users/henry/Projects/personal/vms/node_modules/@nx/js/src/executors/node/node-with-require-overrides"],"memoryUsage":{"rss":131629056,"heapTotal":71008256,"heapUsed":42241512,"external":4365011,"arrayBuffers":141457}},"os":{"loadavg":[2.32080078125,3.01708984375,3.1904296875],"uptime":39672},"trace":[{"column":16,"file":"node:net","function":"Server.setupListenHandle [as _listen2]","line":1898,"method":"setupListenHandle [as _listen2]","native":false},{"column":12,"file":"node:net","function":"listenInCluster","line":1946,"method":null,"native":false},{"column":7,"file":"node:net","function":"Server.listen","line":2044,"method":"listen","native":false},{"column":32,"file":"/Users/henry/Projects/personal/vms/node_modules/@nestjs/platform-express/adapters/express-adapter.js","function":"ExpressAdapter.listen","line":95,"method":"listen","native":false},{"column":30,"file":"/Users/henry/Projects/personal/vms/node_modules/@nestjs/core/nest-application.js","function":null,"line":180,"method":null,"native":false},{"column":null,"file":null,"function":"new Promise","line":null,"method":null,"native":false},{"column":16,"file":"/Users/henry/Projects/personal/vms/node_modules/@nestjs/core/nest-application.js","function":"NestApplication.listen","line":170,"method":"listen","native":false},{"column":21,"file":"/Users/henry/Projects/personal/vms/dist/apps/api/main.js","function":"startServer","line":885,"method":null,"native":false}]}
[31m[App] 14733  - [39m6/24/2024, 9:46:48 PM [31m  ERROR[39m  [31mDBError: password authentication failed for user "dev"
    at wrapError (/Users/henry/Projects/personal/vms/node_modules/db-errors/lib/dbErrors.js:19:14)
    at handleExecuteError (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:1140:32)
    at TableDataBuilder.execute (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:449:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at GetForTable.execute (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5251:24)
    at TableDataBuilder.getForTable (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5263:24)
    at UsersService.findPaging (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5365:16)[39m
[31m[App] 14928  - [39m6/24/2024, 9:47:54 PM [31m  ERROR[39m  [31mDBError: select "users".* from "users" where "0" like $1 limit $2 - column "0" does not exist
    at wrapError (/Users/henry/Projects/personal/vms/node_modules/db-errors/lib/dbErrors.js:19:14)
    at handleExecuteError (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:1140:32)
    at TableDataBuilder.execute (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:449:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at GetForTable.execute (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5251:24)
    at TableDataBuilder.getForTable (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5263:24)
    at UsersService.findPaging (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5365:16)[39m
[31m[App] 15054  - [39m6/24/2024, 9:48:54 PM [31m  ERROR[39m  [31mDBError: select "users".* from "users" where "0" like $1 limit $2 - column "0" does not exist
    at wrapError (/Users/henry/Projects/personal/vms/node_modules/db-errors/lib/dbErrors.js:19:14)
    at handleExecuteError (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:1140:32)
    at TableDataBuilder.execute (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:449:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at GetForTable.execute (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5251:24)
    at TableDataBuilder.getForTable (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5263:24)
    at UsersService.findPaging (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5365:16)[39m
[31m[App] 15667  - [39m6/24/2024, 9:53:04 PM [31m  ERROR[39m  [31mError: getaddrinfo ENOTFOUND DB_HOST
    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)[39m
[31m[App] 15741  - [39m6/24/2024, 9:53:26 PM [31m  ERROR[39m  [31mRangeError: Port should be >= 0 and < 65536. Received type number (NaN).
    at lookupAndConnect (node:net:1297:5)
    at Socket.connect (node:net:1254:5)
    at Socket.connect (/Users/henry/Projects/personal/vms/node_modules/dd-trace/packages/datadog-instrumentations/src/net.js:33:22)
    at Connection.connect (/Users/henry/Projects/personal/vms/node_modules/pg/lib/connection.js:44:17)
    at Client._connect (/Users/henry/Projects/personal/vms/node_modules/pg/lib/client.js:113:11)
    at /Users/henry/Projects/personal/vms/node_modules/pg/lib/client.js:167:12
    at new Promise (<anonymous>)
    at Client.connect (/Users/henry/Projects/personal/vms/node_modules/pg/lib/client.js:166:12)
    at Client_PG._acquireOnlyConnection (/Users/henry/Projects/personal/vms/node_modules/knex/lib/dialects/postgres/index.js:91:23)
    at Client_PG.acquireRawConnection (/Users/henry/Projects/personal/vms/node_modules/knex/lib/dialects/postgres/index.js:99:17)[39m
[31m[App] 15854  - [39m6/24/2024, 9:53:56 PM [31m  ERROR[39m  [31mError: connect ECONNREFUSED 127.0.0.1:6374
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1606:16)[39m
[31m[App] 16007  - [39m6/24/2024, 9:54:41 PM [31m  ERROR[39m  [31mError: connect ECONNREFUSED 127.0.0.1:5433
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1606:16)[39m
[31m[App] 16073  - [39m6/24/2024, 10:00:51 PM [31m  ERROR[39m  [31mDBError: select "users".* from "users" where "0" like $1 limit $2 - column "0" does not exist
    at wrapError (/Users/henry/Projects/personal/vms/node_modules/db-errors/lib/dbErrors.js:19:14)
    at handleExecuteError (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:1140:32)
    at TableDataBuilder.execute (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:449:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at GetForTable.execute (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5251:24)
    at TableDataBuilder.getForTable (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5263:24)
    at UsersService.findPaging (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5365:16)[39m
[31m[App] 16073  - [39m6/24/2024, 10:00:57 PM [31m  ERROR[39m  [31mDBError: select "users".* from "users" where "0" like $1 limit $2 - column "0" does not exist
    at wrapError (/Users/henry/Projects/personal/vms/node_modules/db-errors/lib/dbErrors.js:19:14)
    at handleExecuteError (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:1140:32)
    at TableDataBuilder.execute (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:449:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at GetForTable.execute (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5251:24)
    at TableDataBuilder.getForTable (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5263:24)
    at UsersService.findPaging (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5365:16)[39m
[31m[App] 16073  - [39m6/24/2024, 10:01:07 PM [31m  ERROR[39m  [31mDBError: select "users".* from "users" where "0" like $1 limit $2 - column "0" does not exist
    at wrapError (/Users/henry/Projects/personal/vms/node_modules/db-errors/lib/dbErrors.js:19:14)
    at handleExecuteError (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:1140:32)
    at TableDataBuilder.execute (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:449:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at GetForTable.execute (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5251:24)
    at TableDataBuilder.getForTable (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5263:24)
    at UsersService.findPaging (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5365:16)[39m
[31m[App] 16073  - [39m6/24/2024, 10:01:08 PM [31m  ERROR[39m  [31mDBError: select "users".* from "users" where "0" like $1 limit $2 - column "0" does not exist
    at wrapError (/Users/henry/Projects/personal/vms/node_modules/db-errors/lib/dbErrors.js:19:14)
    at handleExecuteError (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:1140:32)
    at TableDataBuilder.execute (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:449:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at GetForTable.execute (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5251:24)
    at TableDataBuilder.getForTable (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5263:24)
    at UsersService.findPaging (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5365:16)[39m
[31m[App] 16073  - [39m6/24/2024, 10:01:18 PM [31m  ERROR[39m  [31mDBError: select "users".* from "users" where "0" like $1 limit $2 - column "0" does not exist
    at wrapError (/Users/henry/Projects/personal/vms/node_modules/db-errors/lib/dbErrors.js:19:14)
    at handleExecuteError (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:1140:32)
    at TableDataBuilder.execute (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:449:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at GetForTable.execute (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5251:24)
    at TableDataBuilder.getForTable (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5263:24)
    at UsersService.findPaging (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5365:16)[39m
[31m[App] 16073  - [39m6/24/2024, 10:01:19 PM [31m  ERROR[39m  [31mDBError: select "users".* from "users" where "0" like $1 limit $2 - column "0" does not exist
    at wrapError (/Users/henry/Projects/personal/vms/node_modules/db-errors/lib/dbErrors.js:19:14)
    at handleExecuteError (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:1140:32)
    at TableDataBuilder.execute (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:449:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at GetForTable.execute (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5251:24)
    at TableDataBuilder.getForTable (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5263:24)
    at UsersService.findPaging (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5365:16)[39m
[31m[App] 16073  - [39m6/24/2024, 10:01:27 PM [31m  ERROR[39m  [31mDBError: select "users".* from "users" where "0" like $1 limit $2 - column "0" does not exist
    at wrapError (/Users/henry/Projects/personal/vms/node_modules/db-errors/lib/dbErrors.js:19:14)
    at handleExecuteError (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:1140:32)
    at TableDataBuilder.execute (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:449:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at GetForTable.execute (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5251:24)
    at TableDataBuilder.getForTable (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5263:24)
    at UsersService.findPaging (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5365:16)[39m
[31m[App] 16073  - [39m6/24/2024, 10:01:32 PM [31m  ERROR[39m  [31mDBError: select "users".* from "users" where "0" like $1 limit $2 - column "0" does not exist
    at wrapError (/Users/henry/Projects/personal/vms/node_modules/db-errors/lib/dbErrors.js:19:14)
    at handleExecuteError (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:1140:32)
    at TableDataBuilder.execute (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:449:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at GetForTable.execute (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5251:24)
    at TableDataBuilder.getForTable (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5263:24)
    at UsersService.findPaging (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5365:16)[39m
[31m[App] 16073  - [39m6/24/2024, 10:01:33 PM [31m  ERROR[39m  [31mDBError: select "users".* from "users" where "0" like $1 limit $2 - column "0" does not exist
    at wrapError (/Users/henry/Projects/personal/vms/node_modules/db-errors/lib/dbErrors.js:19:14)
    at handleExecuteError (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:1140:32)
    at TableDataBuilder.execute (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:449:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at GetForTable.execute (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5251:24)
    at TableDataBuilder.getForTable (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5263:24)
    at UsersService.findPaging (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5365:16)[39m
[31m[App] 16073  - [39m6/24/2024, 10:01:50 PM [31m  ERROR[39m  [31mDBError: select "users".* from "users" where "0" like $1 limit $2 - column "0" does not exist
    at wrapError (/Users/henry/Projects/personal/vms/node_modules/db-errors/lib/dbErrors.js:19:14)
    at handleExecuteError (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:1140:32)
    at TableDataBuilder.execute (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:449:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at GetForTable.execute (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5251:24)
    at TableDataBuilder.getForTable (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5263:24)
    at UsersService.findPaging (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5365:16)[39m
[31m[App] 18652  - [39m6/24/2024, 10:11:58 PM [31m  ERROR[39m  [31mDBError: select "users".* from "users" where "username" like $1 and "username" like $2 and "0" like $3 limit $4 - column "0" does not exist
    at wrapError (/Users/henry/Projects/personal/vms/node_modules/db-errors/lib/dbErrors.js:19:14)
    at handleExecuteError (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:1140:32)
    at TableDataBuilder.execute (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:449:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at GetForTable.execute (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5251:24)
    at TableDataBuilder.getForTable (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5263:24)
    at UsersService.findPaging (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5365:16)[39m
[31m[App] 18652  - [39m6/24/2024, 10:12:08 PM [31m  ERROR[39m  [31mDBError: select "users".* from "users" where "username" like $1 and "username" like $2 and "0" like $3 limit $4 - column "0" does not exist
    at wrapError (/Users/henry/Projects/personal/vms/node_modules/db-errors/lib/dbErrors.js:19:14)
    at handleExecuteError (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:1140:32)
    at TableDataBuilder.execute (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:449:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at GetForTable.execute (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5251:24)
    at TableDataBuilder.getForTable (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5263:24)
    at UsersService.findPaging (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5365:16)[39m
[31m[App] 18652  - [39m6/24/2024, 10:12:09 PM [31m  ERROR[39m  [31mDBError: select "users".* from "users" where "username" like $1 and "username" like $2 and "0" like $3 limit $4 - column "0" does not exist
    at wrapError (/Users/henry/Projects/personal/vms/node_modules/db-errors/lib/dbErrors.js:19:14)
    at handleExecuteError (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:1140:32)
    at TableDataBuilder.execute (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:449:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at GetForTable.execute (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5251:24)
    at TableDataBuilder.getForTable (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5263:24)
    at UsersService.findPaging (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5365:16)[39m
[31m[App] 18652  - [39m6/24/2024, 10:12:10 PM [31m  ERROR[39m  [31mDBError: select "users".* from "users" where "username" like $1 and "username" like $2 and "0" like $3 limit $4 - column "0" does not exist
    at wrapError (/Users/henry/Projects/personal/vms/node_modules/db-errors/lib/dbErrors.js:19:14)
    at handleExecuteError (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:1140:32)
    at TableDataBuilder.execute (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:449:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at GetForTable.execute (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5251:24)
    at TableDataBuilder.getForTable (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5263:24)
    at UsersService.findPaging (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5365:16)[39m
[31m[App] 18652  - [39m6/24/2024, 10:12:19 PM [31m  ERROR[39m  [31mDBError: select "users".* from "users" where "username" like $1 and "username" like $2 and "0" like $3 limit $4 - column "0" does not exist
    at wrapError (/Users/henry/Projects/personal/vms/node_modules/db-errors/lib/dbErrors.js:19:14)
    at handleExecuteError (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:1140:32)
    at TableDataBuilder.execute (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:449:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at GetForTable.execute (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5251:24)
    at TableDataBuilder.getForTable (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5263:24)
    at UsersService.findPaging (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5365:16)[39m
[31m[App] 20301  - [39m6/24/2024, 10:25:44 PM [31m  ERROR[39m  [31mDBError: select "users".* from "users" where "0" like $1 limit $2 - column "0" does not exist
    at wrapError (/Users/henry/Projects/personal/vms/node_modules/db-errors/lib/dbErrors.js:19:14)
    at handleExecuteError (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:1140:32)
    at TableDataBuilder.execute (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:449:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at GetForTable.execute (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5251:24)
    at TableDataBuilder.getForTable (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5263:24)
    at UsersService.findPaging (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5366:16)[39m
[31m[App] 20301  - [39m6/24/2024, 10:26:20 PM [31m  ERROR[39m  [31mDBError: select "users".* from "users" where "0" like $1 and "username" like $2 limit $3 - column "0" does not exist
    at wrapError (/Users/henry/Projects/personal/vms/node_modules/db-errors/lib/dbErrors.js:19:14)
    at handleExecuteError (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:1140:32)
    at TableDataBuilder.execute (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:449:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at GetForTable.execute (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5251:24)
    at TableDataBuilder.getForTable (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5263:24)
    at UsersService.findPaging (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5366:16)[39m
[31m[App] 20301  - [39m6/24/2024, 10:26:31 PM [31m  ERROR[39m  [31mDBError: select "users".* from "users" where "0" like $1 and "username" like $2 and "username" like $3 limit $4 - column "0" does not exist
    at wrapError (/Users/henry/Projects/personal/vms/node_modules/db-errors/lib/dbErrors.js:19:14)
    at handleExecuteError (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:1140:32)
    at TableDataBuilder.execute (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:449:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at GetForTable.execute (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5251:24)
    at TableDataBuilder.getForTable (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5263:24)
    at UsersService.findPaging (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5366:16)[39m
[31m[App] 23118  - [39m6/24/2024, 10:52:56 PM [31m  ERROR[39m  [31mDBError: password authentication failed for user "mq"
    at wrapError (/Users/henry/Projects/personal/vms/node_modules/db-errors/lib/dbErrors.js:19:14)
    at handleExecuteError (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:1140:32)
    at TableDataBuilder.execute (/Users/henry/Projects/personal/vms/node_modules/objection/lib/queryBuilder/QueryBuilder.js:449:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at GetForTable.execute (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5253:24)
    at TableDataBuilder.getForTable (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5266:24)
    at UsersService.findPaging (/Users/henry/Projects/personal/vms/dist/apps/api/main.js:5369:16)[39m
